website/source/docs/guides/acl-legacy.html.md

```diff
@@ -95,7 +95,7 @@ Constructing rules from these policies is covered in detail in the
 #### ACL Datacenter
 
 All nodes (clients and servers) must be configured with a
-[`primary_datacenter`](/docs/agent/options.html#primary_datacenter) which enables ACL
+[`acl_datacenter`](/docs/agent/options.html#acl_datacenter) which enables ACL
```

Comment:
I can see the rationale for reverting this in the legacy docs for older versions. Do you think we need a note at the top though about how it's using the old, deprecated config syntax?

I guess in practice we'll keep this syntax as long as we keep old ACL support (which is likely ages) so not super important.

Reply:
@banks : Adding a note would be beneficial as long as we're supporting the Legacy ACL syntax